### PR TITLE
Orbital drop pods

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

Sorta what it says on the tin
Has a control console to launch a bunch at once
Also cleaned up some tgui while I was there

todo:

- [x] map it in
- [x] logging¨

- [x] make them onl launchable after alamo leaves?

- [x] wait for weather pr and add a outside check

## Why It's Good For The Game

Kewl

## Changelog
:cl: Tiviplus, Mapping by Hathkar
add: Zeus orbital Droppods
/:cl:

### CODE REVIEW

**Comment by jroinc** [comment]
From tests a while back, using these to deploy directly into combat in way is suicide due to unsupported ungas with no fortifications being torn to shreds.

Is this just gonna be a "fast travel" system to the FOB/ungaball?

**Comment by TiviPlus** [comment]
by "testing" I assume you mean "admins teleport a few marines to a random location on the map at 12:20 and they die because marines havent even gotten the FOB done yet so they can come help"

This offers proper control over who goes where and when and thus wont rely on an admin pushing buttons

**Review by TheBimmer231 — APPROVED** [review]
Good sprite, approved.

**Comment by AtlasRules** [comment]
finally, we can have MULTIPLE, SIMILTANIOUS, DEFENSIVE DEVASTATING DEEP STRIKES

**Comment by davidchan13** [comment]
THE CODEX RETARDIS IDENTIFIES THIS MANEUVER AS TEAL RAHNE

File: code/game/objects/machinery/computer/droppod_control.dm [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/obj/machinery/computer/droppod_control
+	name = "Droppod launch computer"
+	desc = "A personal computer used to view emails"
```
Comment (Hughgent) [inline code/game/objects/machinery/computer/droppod_control.dm:3]: This needs a 'better' description.

**Review by Hughgent — APPROVED** [review]

File: code/game/objects/structures/droppod.dm [hunk 2]
```diff
@@ -0,0 +1,216 @@
+#define DROPPOD_READY 1
+#define DROPPOD_ACTIVE 2
+#define DROPPOD_LANDED 3
+///Marine steel rain style drop pods, very cool!
+/obj/structure/droppod
+	name = "TGMC Zeus Orbital drop pod"
```
Comment (XSlayer300) [inline code/game/objects/structures/droppod.dm:6]:
```suggestion
	name = "TGMC Zeus orbital drop pod"
```

**Comment by Hughgent** [comment]
24 hours of only testing this and no bug reports/issues raised.

**Pull request merged**